spec/factories.rb

```diff
@@ -25,10 +25,20 @@
     expires_at { 2.minutes.from_now }
   end
 
-  factory :assessments_assessment, class: Assessments::Assessment do
-    api_name { Faker::Lorem.word }
-    timestamp { Time.now }
+  factory :assessment_log do
+    incognia_id { SecureRandom.uuid }
 
-    initialize_with { new(attributes) }
+    trait :onboarding do
+      api_name { :onboarding }
+      incognia_signup_id { SecureRandom.uuid }
+    end
+
+    AssessmentLog.api_names.except(:onboarding).values.each do |api_name|
```

Comment:
I put this in some places to make it easy when new api_names appear at `AssessmentLog`, but it is a little cryptic. :all_good.

Do you guys think it is worth it?

Reply:
Depending on what you say, I can refactor this line of code to be centralized and use at tests.

Reply:
I prefer centrailizing it, but I didn't quite understand this line.
Whenever we define a new `api_name` wouldn't it have its own "uniqueness" which would lead us to building a specific trait or specific test context as `onboarding`?

Reply:
I tried to simplify without entirely losing the opportunity of reuse in case of a newer `api_name`.

fd8f8cacd0d88f53f04dc4785d9cf93bb9b64647

Reply:
Ops! Needed to amend, the correct one is here: d5cdd7e000707e47c5a2f798fa571adf70fca6d5